tests/test_presolve.py

```diff
@@ -201,8 +211,8 @@ def test_no_variable_constraints(self):
 
             presolver = Presolver(cqm)
             presolver.load_default_presolvers()
-            with self.assertRaises(InfeasibleModelError):
-                presolver.apply()
+            presolver.apply()
+            self.assertIs(presolver.feasibility(), Feasibility.Infeasible)
 
         with self.subTest("infeas >="):
```

Comment:
would also be good to have a test for feasible model.

Reply:
We do have a bunch of tests for that at the C++ level